docs/install/any-kubernetes-cluster.md

```diff
@@ -717,7 +717,29 @@ At this point, you have a basic installation of Knative Eventing!
 
    <!-- This indentation is important for things to render properly. -->
 
-{{< tabs name="eventing_extensions" >}} {{% tab name="Sugar Controller" %}}
+{{< tabs name="eventing_extensions" >}}
```

Comment:
Does this really belong here? It feels like we should keep the eventing install more basic than this, since Kafka is more of an integration than something out of the box like Knative services or other components as sinks?

Reply:
@RichieEscarez any thoughts on this?
cc @knative/eventing-wg-leads

Reply:
I agree, there is not enough context here about what all those "extensions" do and should be introduced in a separate page.

Also, the section immediately following this ("Getting started with Eventing") is a more intuitive path for users to take and basically duplicates those options.

It looks like this was already published in 0.19 and this section should also be removed from there too.

![image](https://user-images.githubusercontent.com/11762685/101392809-0b7ff480-387b-11eb-92aa-b1d8c97696bc.png)

Reply:
I followed what was already there.